lib/node_modules/@stdlib/stats/base/dists/gumbel/skewness/src/main.c

```diff
@@ -0,0 +1,46 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/sqrt.h"
+#include "stdlib/math/base/special/riemann_zeta.h"
+#include "stdlib/constants/float64/pi.h"
+
+/**
+* Returns the skewness for a Gumbel distribution with location `mu` and scale `beta`.
+*
+* @param mu     location parameter
+* @param beta   scale parameter
+* @returns      skewness
+*
+* @example
+* double y = stdlib_base_dists_gumbel_skewness( 0.0, 1.0 );
+* // returns ~1.14
+*/
+double stdlib_base_dists_gumbel_skewness( const double mu, const double beta ) {
+	double SKEWNESS = 12.0 * stdlib_base_sqrt( 6.0 ) * stdlib_base_zeta( 3.0 ) / ( STDLIB_CONSTANT_FLOAT64_PI*STDLIB_CONSTANT_FLOAT64_PI*STDLIB_CONSTANT_FLOAT64_PI );
```

Comment:
Noticing that this constant can be precomputed and moved to the outer scope as a `static const double`. This way, we can also avoid depending on the `sqrt` and `zeta` functions here. A similar change should be made to the JS implementation, if applicable.